backend/LexData/Migrations/20240417115559_Add IsSensitive column to Project and DraftProject.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace LexData.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddIsSensitivecolumntoProjectandDraftProject : Migration
```

Comment:
```suggestion
    public partial class AddIsSensitiveColumnToProjectandDraftProject : Migration
```
No idea if this class name should be changed in other places - sort of looks like it is consumed automagically

Reply:
It's created automatically from the parameter to `task api:add-migration`, but the name doesn't truly matter: as long as it's changed in all the places where the partial class is defined (two files, this one and the `_Designer.cs` file) then the name change shouldn't matter.

What you should *not* change is the `[Migration("20240417115559_Add IsSensitive column to Project and DraftProject")]` attribute in the `_Designer.cs` file. That one needs to be the same string that gets stored in the database. But as long as that string is the same, you could rename the classes to FooMigration and it would still work.

Reply:
Actually this may become a moot point: if we change IsSensitive to IsConfidential, you'll probably want to abandon this migration with `task api:remove-last-migration` and create a new one that creates an IsSensitive column instead.